api/app.py

```diff
@@ -88,7 +88,7 @@ def get_data():
         raise APIException("Missing JSON body or proper mimetype")
 
     q = body["query"]
-
+    print(q)
```

Comment:
strange – this was removed in a recent commit but does not appear to be rflected here